dev-docs/modules/jwplayerRtdProvider.md

```diff
@@ -31,33 +31,24 @@ to Bid Adapters.
 2) Publishers must register JW Player as a Real Time Data provider by using `setConfig` to load a Prebid Config containing a `realTimeData.dataProviders` array:
 
 ```javascript
-const jwplayerRtdProvider = {
-  name: "jwplayer"
-};
-
 pbjs.setConfig({
     ...,
     realTimeData: {
       auctionDelay: 100,
-      dataProviders: [
-          jwplayerRtdProvider
-      ]
+      dataProviders: [{
+          name: "jwplayer",
+          waitForIt: true,
+          params: {
+            mediaIDs: ['abc', 'def', 'ghi', 'jkl']
+          }
+      }]
     }
 });
 ``` 
 
-3) In order to prefetch targeting information for certain media, include the media IDs in the `jwplayerRtdProvider` var and set `waitForIt` to `true` before calling `setConfig`:
+3) In order to prefetch targeting information for certain media, include the media IDs in the `jwplayer` var and set `waitForIt` to `true` before calling `setConfig`:
```

Comment:
i disagree with this change, there is no `jwplayer` var. I think this would be confusing to the reader. 
`var jwplayer = {...};` would be a jwplayer var. I'll address this